sdk/containerregistry/azure-containerregistry/azure/containerregistry/__init__.py

```diff
@@ -8,6 +8,7 @@
 
 from ._container_registry_client import ContainerRegistryClient
 from ._container_repository_client import ContainerRepositoryClient
+from ._exchange_client import ACRExchangeClient
```

Comment:
This isn't added to `__all__`?
Do we expect customers to use this directly? Or do we only use it internally in the credential implementation?

Reply:
This is used internally for credential implementation, I do not expect customers to use it directly.